python/tritonserver/_c/tritonserver_pybind.cc

```diff
@@ -1159,6 +1162,11 @@ class PyInferenceRequest
     ThrowIfError(TRITONSERVER_InferenceRequestSetBoolParameter(
         triton_object_, key.c_str(), value));
   }
+  void SetDoubleParameter(const std::string& key, double value)
+  {
+    ThrowIfError(TRITONSERVER_InferenceRequestSetDoubleParameter(
```

Comment:
nit: `THROW_IF_ERROR`

Reply:
we usually use all caps only for macros and constants - in this case ThrowIf Error is an actual function -

Reply:
Ah, I see, thanks for explaining! I thought it is a macro.